.github/workflows/ci.yml

```diff
@@ -188,6 +188,16 @@ jobs:
           toolchain: 1.57.0
       - run: cargo check --lib --all-features
 
+  cross:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          persist-credentials: false
+      - uses: dtolnay/rust-toolchain@nightly
+      - run: cargo install cross
```

Comment:
Question: This seems to add about a minute to the CI runtime. Do you think its worth caching the install or using [cargo-binstall](https://github.com/cargo-bins/cargo-binstall) for a pre-compiled version?

Reply:
I was hesitant to add any 3rd party actions since I think the project has taken a pretty conservative stance on those in the past.

I'm not familiar with `cargo-binstall` but it does sound interesting. Maybe we could consider the speed optimizations in a follow-up once the breakage is addressed?

Reply:
Looks like https://github.com/marketplace/actions/install-development-tools would maybe be the best way to integrate (?)

Reply:
The "is this secure" part of the FAQ (https://github.com/cargo-bins/cargo-binstall#faq) gives me some pause :-/

Reply:
Their description doesn't seem horribly problematic (they go from crates.io -> GitHub), but its probably not worth the 1 minute of CI time then if we're both slightly hesitant. I was hoping there was a more clear answer for pre-compiled binaries.